pkg/summary/summarizers.go

```diff
@@ -135,6 +136,7 @@ func init() {
 	}
 
 	Summarizers = []Summarizer{
+		checkGeneration,
```

Comment:
I thought the [issue pointed at in this comment](https://github.com/rancher/rancher/issues/49671#issuecomment-2864284813) was that there was no `.status` since the object was created. How does looking at observedGeneration and metadata generation solves this?

I understand that observedGeneration can be used to know if the status is stale by comparing metadata generation.